plan9/header.patch

```diff
@@ -0,0 +1,11 @@
+--- curses.h	2020-06-04 13:06:57.089157964 +0200
++++ curses.ape	2020-06-05 11:17:57.562007451 +0200
+@@ -27,6 +27,8 @@
+ 
+ **man-end****************************************************************/
+ 
++#pragma lib "/$M/lib/ape/libcurses.a"
++
+ #define PDCURSES        1
```

Comment:
The version numbers are part of the curses header (and different), what would be the reason to extra define them?

Reply:
Indeed the patch was made against upstream PDCurses and I just imported the code thinking that the forks would not differ too much. A bit naive of me. I will make a number of changes and make sure that it builds also on this fork.